components/tabs/tab-internal.js

```diff
@@ -90,50 +98,71 @@ class Tab extends RtlMixin(LitElement) {
 			}
 		`;
 	}
-
+	
 	constructor() {
 		super();
-		this.ariaSelected = 'false';
-		this.role = 'tab';
-		this.tabIndex = -1;
-	}
-
-	firstUpdated(changedProperties) {
-		super.firstUpdated(changedProperties);
-		this.addEventListener('click', () => {
-			this.ariaSelected = 'true';
-		});
-		this.addEventListener('keydown', (e) => {
-			if (e.keyCode !== keyCodes.SPACE) return;
-			e.stopPropagation();
-			e.preventDefault();
-		});
-		this.addEventListener('keyup', (e) => {
-			if (e.keyCode !== keyCodes.ENTER && e.keyCode !== keyCodes.SPACE) return;
-			this.ariaSelected = 'true';
-		});
+		this.selected = false;
+		this.role = 'none';
 	}
 
 	render() {
-		return html`
+		return this.href ? html`
+		<a
+		 href="${this.href}"
+		 role="tab"
+		 tabindex="${this.activeFocusable ? 0 : -1}"
+		 aria-selected="${this.selected}"
+		 aria-controls="${this.controlsPanel}"
```

Comment:
This currently doesn't work because of shadow DOM, but `aria-controls` isn't used currently for `d2l-tabs` anyway. I have another branch that doesn't use shadow DOM where this would work.